designcompose/src/main/java/com/android/designcompose/Utils.kt

```diff
@@ -127,14 +91,14 @@ internal fun absoluteLayout(style: ViewStyle, constraints: Constraints, density:
             0
         }
 
-    val left = style.layout_style.left.resolve(pw, density)
-    val top = style.layout_style.top.resolve(ph, density)
+    val left = style.layout_style.left.get().resolve(pw, density)
```

Comment:
I'm confused because I thought `get()` would return `Dimension`, but you changed `resolve()` to be a function on `DimentionProto`, not `Dimension`.